### DISCUSSION TITLE
Just wondering how this works - see body

### DISCUSSION THREAD

**Discussion by con-dog** [Glench/ExtPay#178] [Q&A] [answered]
So does this require a user to have a chrome login?

Of is it just tied to an input email? Eg user enters their email, they get a password link etc.

I'f love to know how it works and the limitations before trying it

**Comment by SamWadrose** [comment, marked as answer]
All this requires for a sign in is a user email. A link is emailed to that user which then grants them access to sign in once the click it. No password or chrome authentication required.
